packages/probing-frontend/src/layout/LoginLayout.tsx

```diff
@@ -0,0 +1,48 @@
+import { Paper, Typography } from '@mui/material'
+import { Box } from '@mui/system'
+import type { KeyPrefix } from 'i18next'
+import { useTranslation } from 'react-i18next'
+
+export const LoginLayout = ({
+  children,
+  i18nContext,
+}: {
+  children: React.ReactNode
+  i18nContext: KeyPrefix<'common'>
```

Comment:
It's always better to keep the component dependencies (props) as simple as possible for testing purposes. I would transform the `i18nContext` prop to an explicit `title` and `description` props.